src/components/feature-board/CardHost.vue

```diff
@@ -21,14 +21,13 @@
 </template>
 
 <script lang="ts">
-import { defineComponent } from "vue";
+import { defineComponent, onUpdated } from "vue";
```

Comment:
```suggestion
import { defineComponent } from "vue";
```
onUpdated is not used, so we can remove it